pkg/telepresence/v2/bin/intercept

```diff
@@ -15,21 +15,33 @@ trap leave EXIT
 
 i=3 # account for $0, $workflow and "--" argument
 
+ctx=$(kubectl config current-context)
+ns=$(kubectl config view --minify -o jsonpath='{..namespace}')
+
 for port in ${@:2}; do
     if [[ "$port" == "--" ]]; then
         break
     fi
 
-    name="${workload}_${port}"
+    name="${ctx}_${ns}_${workload}_${port}"
     local_port="$((32768 + $RANDOM))"
 
-    telepresence intercept \
+    telepresence \
+        --use "$name" \
+        connect \
+        --name "$name" \
+        --context "$ctx" \
+        --namespace "$ns"
+    telepresence \
+        --use "$name" \
+        intercept \
         "$name" \
         --workload "$workload" \
         --port "${local_port}:${port}" \
         --env-file="${env_file}"
 
-    echo "telepresence leave '${name}'" >> "$defer"
+    echo "telepresence --use '${name}' connect --name '${name}' --context '${ctx}' --namespace '${ns}'" >> "$defer"
```

Comment:
What's the purpose of this line?

Reply:
You have to `telepresence connect` before you can `telepresence leave`. It will give the same warning as `telepresence intercept`, if you've had an intercept open long enough.